backend/src/main/java/com/redhat/cloud/notifications/db/Query.java

```diff
@@ -158,8 +174,9 @@ public Optional<Sort> getSort() {
         }
 
         if (sortFields == null) {
-            Log.warnf("NOTIF-674 SortFields not set.");
-            sortFields = Map.of();
+            InternalServerErrorException isee = new InternalServerErrorException("SortFields are not set");
+            Sentry.captureException(isee);
```

Comment:
Is this needed because RESTEasy prevents the exception from being written in the log? If so, is `Sentry.captureException` still needed if we throw a `RuntimeException` instead of an `InternalServerErrorException`?

Reply:
> Is this needed because RESTEasy prevents the exception from being written in the log?

Yes, that is my understanding.

>  If so, is `Sentry.captureException` still needed if we throw a `RuntimeException` instead of an `InternalServerErrorException`?

I guess that would work - is there any preference over `Sentry.captureException` ?

Reply:
`Sentry.captureException` is less standard/obvious than throwing a simple `RuntimeException`. I'm okay with using `Sentry.captureException` as long as there's a comment explaining what it does and why we need it 😃

Reply:
Updated it - instead of using `Sentry.capture` I'm simply logging an error.